### ISSUE TITLE
Repeated buttons near the search bar on the NFT details page.

### ISSUE THREAD

**Issue by poor-defined** [Magickbase/ckb-explorer-public-issues#476]
![image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/85155313/bd0e59c8-16ae-4f01-a168-2686b0f1aefa)

**Comment by Kirl70** [comment]
This button is used to reset the search box after searching, and it serves the same function on other pages as well. Should it be removed from the other pages too?

**Comment by Sven-TBD** [comment]
Currently, the function of this button is no the same with that. It can be changed to be consistent, that is, when the filter setting is inconsistent with default, clear the filter setting.

**Comment by Kirl70** [comment]
So, the reset button on the NFT details page should be kept, right?

**Comment by Sven-TBD** [comment]
Yes, just adjust its function without modifying the UI part.

**Comment by Sven-TBD** [comment]
Reset function should be defined as reset the filter config and then refresh page。

**Comment by Daryl-L** [comment]
But in some other pages, no filter button exists. Like the [https://explorer.nervos.org/block/11490733](block detail page).

> Because search and filtering are separate, when users use filtering, the search function becomes inactive.
> 
> Therefore, the reset button interaction here can be modified as follows:
> 
> 1.Clear filter when using search 2.Display the reset button only when some filters are checked.
> 
> 由于搜索和筛选是分开的。所以当用户使用筛选时，搜索即失效了。
> 
> 因此，这里的reset按钮交互可以修改为： 1 .当使用搜索时，清空filter 条件 2. 当filter被采用时，才出现reset按钮

**Comment by Sven-TBD** [comment]
I see... Then I think we can move it inside the filter box @Kirl70 @Daryl-L

**Comment by Kirl70** [comment]
Do you mean to place the reset option inside the filter popup?
<img width="366" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/106060898/aa34200d-4718-4f6f-829f-1fa56b29ab23">

**Comment by Sven-TBD** [comment]
> Do you mean to place the reset option inside the filter popup? <img alt="image" width="366" src="https://private-user-images.githubusercontent.com/106060898/294444467-aa34200d-4718-4f6f-829f-1fa56b29ab23.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0NTM4MDMsIm5iZiI6MTcwNDQ1MzUwMywicGF0aCI6Ii8xMDYwNjA4OTgvMjk0NDQ0NDY3LWFhMzQyMDBkLTQ3MTgtNGY2Zi04MjlmLTFmYTU2YjI5YWIyMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwNVQxMTE4MjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMDFiZWYxODVjOGE0YmFhNjhiOGM2ZjdjZTU3MDY5NzdhMjM0MDI4ZDRlNDE5NjY1NThlYTI1M2NjNzhjZWUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yHzSSliFJodsg8bqZAREFE7yRDL4LgVX6NeJ9fgBeII">

Yes, and maybe it could be a little bit different from other options.

**Comment by Kirl70** [comment]
> Yes, and maybe it could be a little bit different from other options.

Perhaps the reset action could be performed on the filter icon.
<img width="340" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/106060898/033c837e-d287-4ffe-893e-451d017b0155">
